e2e/porch_test.go

```diff
@@ -88,6 +89,10 @@ func runTestCase(t *testing.T, git string, tc porch.TestCaseConfig) {
 
 		err := cmd.Run()
 
+		if command.Yaml {
+			reorderYamlStdout(t, &stdout)
```

Comment:
Do we want the command to render a stable output ?

Reply:
This would likely require changes to kyaml writers because they are currently in use and they generate the unstable output.
Unless the contract on ResourceList defines ordering (which I don't think it does) I would rather leave the primary implementation as-is to not make it adhere to a different contract, and have the test compensate for order issues.

That said I assume that dealing with ordering is something kpt tests must do already (for example test for `kpt fn source` ... I can look how the ordering issues are handled there.